src/plugins/discourse/weights.test.js

```diff
@@ -32,6 +60,46 @@ describe("plugins/discourse/weights", () => {
     });
   });
 
+  describe("_categoryWeight", () => {
+    it("returns the default weight when the category doesn't have a configured weight", () => {
+      expect(_categoryWeight(1, getWeights(getConfig()))).toBe(1);
+    });
+    it("returns the default weight when the category doesn't have a configured weight", () => {
+      const config = getConfig({
+        defaultCategoryWeight: 1,
+        categoryWeights: {"1": 5},
+      });
+      expect(_categoryWeight(1, getWeights(config))).toBe(5);
+    });
+    it("can return a weight set to 0", () => {
+      const config = getConfig({
+        defaultCategoryWeight: 1,
+        categoryWeights: {"1": 0},
+      });
+      expect(_categoryWeight(1, getWeights(config))).toBe(0);
+    });
+  });
+
+  describe("_tagWeight", () => {
+    it("returns the default weight when the category doesn't have a configured weight", () => {
+      expect(_weightFromTags(["tag"], getWeights(getConfig()))).toBe(1);
+    });
```

Comment:
s/category/tag, also the test says it checks that we return the default weight, but it actually checks if it returns 1 -- ie it doesn't test what happens if the default weight is not 1. please fix